lux/test/unit/lux/lenses/discord/messages/read_message_lens_test.exs

```diff
@@ -0,0 +1,110 @@
+defmodule Lux.Lenses.Discord.Messages.ReadMessageLensTest do
+  @moduledoc """
+  Test suite for the ReadMessageLens module.
+  These tests verify the lens's ability to:
+  - Read messages from Discord channels
+  - Handle Discord API errors appropriately
+  - Validate input/output schemas
+  """
+
+  use UnitAPICase, async: true
+  alias Lux.Lenses.Discord.Messages.ReadMessageLens
+
+  @channel_id "123456789012345678"
+  @message_id "987654321098765432"
+
+  setup do
+    Req.Test.verify_on_exit!()
```

Comment:
No point using `verify_on_exit!` if we do not use `expect`. (PS: we should use `expect`)